tests/unit/server/bundle_store_test.py

```diff
@@ -32,12 +44,10 @@ def test_bundle_store_workflow(self):
         bundle_stores = self.bundle_manager._model.get_bundle_stores(self.user_id)
         self.assertEqual(len(bundle_stores), 1)
         self.assertEqual(bundle_stores[0].get("name"), "im-not-a-store")
-        # delete the store
+        # Deletion should succeed since there are bundle locations associated with the bundle store
```

Comment:
@adiprerepa I think your test had the opposite behavior / reverse logic, I had to reverse the logic here to get it to work correctly.